accepted/2020/workloads/workloads.md

```diff
@@ -0,0 +1,92 @@
+* `dotnet bundle update [bundle | project]` -- Updates a bundle, in terms of a project, by name, or as part of updating all known bundles.
+
+We don't intend for the .NET CLI-based installation to be mix-and-matched with a package tool (like APT). Instead, we are thinking that an installation tool would provide a mapping file that mapped its packages to workloads, so that `dotnet bunble restore` could provide meaningful error messages that direct users to install workloads via the installation tools. We still need to spend more time defining interactions between the .NET CLI and an installation tools.
+
+In addition to being optional, workloads will be updatable. The SDK will include a mechanism to discover if a workload manifest pack has been updated. An updated manifest could represent bug fixes or new features (like updated bindings for an operating system update). If a manifest has been updated, the user will be notified, and have the opportunity to acquire the new version. Updates will not be automatic. This is roughly similar to the combination of `apt-get update` and `apt-get upgrade` with the [Debian package manager](https://en.wikipedia.org/wiki/APT_(software)).
+
+### SDK and workload versioning
+
+Another benefit of the existing monolithic SDK is that the workloads it includes are known to be compatible with core SDK components, like the C# compiler, NuGet, MSBuild, and the SDK targets. Over time, we have developed the concept of an SDK version train. These are represented by SDK version hundreds bands like `3.1.100` and `3.1.200`. These version trains were created to align with Visual Studio versions, like `16.4` and `16.5`. We do this because those same core SDK components are shared between Visual Studio and the .NET SDK, and need to stay in alignment. In addition, new major versions of those components, possibly containing a new language version, can be delivered in these updates. We expect that workload manifests will need to be re-published for each SDK version train. That's an unfortunate requirement, however, the SDK version trains are a compatibility boundary that we would like to maintain.
+
+## Timing
+
+This project has a very large scope, and requires changing how multiple large bodies of software work. We are planning the following multi-release progression to phase the work.
+
+* .NET 5.0
+  * Define manifest format and packaging.
+  * Enable Xamarin and Web assembly as workloads.
+  * Distribute these workloads only via Visual Studio (Windows and Mac).
+  * Expose few if any .NET CLI verbs for acquisition.
+* .NET 6.0+
+  * Make WPF, Windows Forms and ASP.NET Core as workloads
+  * Enable acquisition of all workloads via the .NET CLI, and at least ASP.NET Core and Web Assembly workloads available via Linux package managers.
+  * Expose the full set of .NET CLI verbs for acquisition.
```

Comment:
Can you combine this with the above?

Reply:
I see the relation, but they are logically separate (not even done by the same teams).